coalaip_bigchaindb/plugin.py

```diff
@@ -1 +1,114 @@
-# -*- coding: utf-8 -*-
+from bigchaindb_driver import BigchainDB
+from bigchaindb_driver.crypto import generate_keypair
+
+
+# FIXME: maybe this should inherit from a base plugin class provided by pycoalaip?
+
+class Plugin:
+    """BigchainDB ledger plugin for COALA IP's Python reference
+    implementation (LINK).
+
+    Plugs in a BigchainDB instance as the persistence layer for COALA IP
+    related actions.
+    """
+
+    def __init__(self, *nodes):
+        """Initialize a :class:`~coalaip_bigchaindb.Plugin` instance
+        and connect to one or more BigchainDB nodes.
+
+        Args:
+            *nodes (str): one or more URLs of BigchainDB nodes to
+                connect to as the persistence layer
+        """
+
+        self.driver = BigchainDB(*nodes)
+
+    def create_user(self):
+        """Create a new verifying/signing keypair for use with BigchainDB
+
+        Returns:
+            tuple: a new verifying key ('verifying_key') and signing key
+                ('signing_key') pair.
+        """
+
+        return generate_keypair()
+
+    def get_status(self, persist_id):
```

Comment:
Should the plugin really expose these very BigchainDB specific things?

I feel like we could boil it down to a fewer statues at one point, for example:

The asset was either persistent, or it wasn't.